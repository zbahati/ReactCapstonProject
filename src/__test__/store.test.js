import store from '../redux/store';

describe('Store', () => {
  test('store matches the snapshot', () => {
    expect(store.getState()).toMatchSnapshot();
  });
});
