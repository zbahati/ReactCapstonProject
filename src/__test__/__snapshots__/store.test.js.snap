// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store store matches the snapshot 1`] = `
Object {
  "countries": Object {
    "countriesData": Array [],
    "countryDetailsData": Array [],
    "loading": false,
    "message": "",
    "searchName": "",
  },
}
`;
